### ISSUE TITLE
Can't run 

### ISSUE THREAD

**Issue by OmerElshrief** [lsyads/fisheye-ORB-SLAM#3]
Hello, 
whenever I try to run the example 
`!./Examples/Monocular/mono_tum Vocabulary/ORBvoc.bin Examples/Monocular/fisheye_eucm_cam0.yaml path`
the CMD just keep loading and nothing happen, even If I enter false path same thing goes on.

**Comment by JiasenTian** [comment]
Yeah, same results for me. Do you have any ideas with it? Have you solved it?

**Comment by JiasenTian** [comment]
> Hello,
> whenever I try to run the example
> `!./Examples/Monocular/mono_tum Vocabulary/ORBvoc.bin Examples/Monocular/fisheye_eucm_cam0.yaml path`
> the CMD just keep loading and nothing happen, even If I enter false path same thing goes on.

Have you solved it? Same result for me.

**Comment by viettoan151** [comment]
Hello all, @OmerElshrief  @JiasenTian ,
I confirm its worked.
Please check the code of _Examples/Monocular/mono_tum.cc_, in main function, check the line:
```c++
string strFile = string(argv[3])+"/rgb.txt";
```
It requires a file of list images  _rgb.txt_ under your input folder path.
The format of _rgb.txt_ is 
```text
1520525725122645511 data/1520525725122645511.png
1520525725372653511 data/1520525725372653511.png
1520525725622662511 data/1520525725622662511.png
1520525725872669511 data/1520525725872669511.png
1520525726122677511 data/1520525726122677511.png
...
```

**Comment by Showmak1r** [comment]
> 

Thanks a lot, bro
